src/components/react/ChessStudy.tsx

```diff
@@ -56,19 +56,15 @@ export const ChessStudy = ({
 	dataAdapter,
 }: AppProps) => {
 	// Parse Obsidian / Code Block Settings
-	const { boardColor, boardOrientation, fen, viewComments, chessStudyId } =
-
+	const { boardColor, boardOrientation, viewComments, chessStudyId } =
 		parseUserConfig(pluginSettings, source);
 
 	// Setup Chessground API
 	const [chessView, setChessView] = useState<Api | null>(null);
 
 	// Setup Chess.js API
-	const [firstPlayer, initialMoveNumber, initialChessLogic] = useMemo(() => {
-		const chess = (fen) ? new Chess(fen) : new Chess();
```

Comment:
Great idea - I will have time on Saturday again to work on this and then release the new version

Reply:
I'm around today as well. Whenever we get this PR through, I'll update my dev branch and start sending you additional PRs. I've posted an image of the current state of my dev branch. I'm really happy with how it has come together so far. Obviously, you can reject or change whatever you want. It has two buttons to tag and annotate moves. You can edit variations to create duplicate variations with alternative moves. It can export the PGN with the [FEN tag] and supports creating a board from that kind of PGN. You can cycle to the root position of the board. You can cycle from the last position to the first. You can pick which move you want to save as the current move. It has a few other bug fixes mixed in there as well. I never did get coordinates to work though. Those have been stubborn.

![image](https://github.com/chrislicodes/obsidian-chess-study/assets/85068850/0078f3da-998f-4123-bd10-34303fb40c38)

Reply:
Ha, nice, super exciting! Cannot wait to check that out. Also I have to try out that combination with the plugin, I just recently got back into Chess. I will ping you tomorrow once the PR is merged and then we can bring in the other features. 🔥